src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/EmailContent.java

```diff
@@ -11,10 +14,12 @@
         private String originatingAppId;
         private String messageId;
         private String messageType;
-        private Map<String, Object> data;
```

Comment:
Small thing but I would suggest to keep the builder type as a Map, and only change it to a string on build()
In my head I wouldn't expect a "with" method to be transforming any data

Reply:
Moved this to build have had som make some changes to the mdoel to accommodate this.